recipes-soletta/dev-app/soletta-dev-app_git.bb

```diff
@@ -2,12 +2,12 @@ DESCRIPTION = "Soletta Development Application"
 RDEPENDS_${PN} = "soletta nodejs systemd graphviz libmicrohttpd avahi-daemon bash git"
 LICENSE = "Apache-2.0"
 
-PV = "1_beta10"
+PV = "1_beta11"
 
 LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=93888867ace35ffec2c845ea90b2e16b"
-SRC_URI[md5sum] = "00f1c0b970cc8d682c7a20c960f4b8e7"
-SRC_URI[sha256sum] = "98ab16895a65692221035998a73d830b04a6835c672c9d64cac2010431519f8e"
-SRC_URI = "https://github.com/solettaproject/soletta-dev-app/releases/download/v${PV}/soletta-dev-app_standalone_v${PV}.tar.gz;protocol=archive \
+SRC_URI[md5sum] = "9c4dfd30cad368c3d57542eff288d33d"
+SRC_URI[sha256sum] = "8bfd68dd20fd8bb77c9080600f3ae36e13ea537f31ebf335eb83a6e0dec6c80b"
+SRC_URI = "https://github.com/solettaproject/soletta-dev-app/releases/download/v${PV}/soletta-dev-app_standalone_v${PV}-2.tar.gz;protocol=archive \
```

Comment:
Now there is a single package for v1_beta11. The "-2" one was for tests, to validade the release generation script was working fine.